lib/averages.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {total, mean}
```

Comment:
It's more common practice to keep your modules.exports at the bottom of your file